Subject: xlf.command.not.found
From: Michel Normand <normand@fr.ibm.com>

try to bypass error while building ppc64le
"make[2]: xlf: Command not found"

Signed-off-by: Michel Normand <normand@fr.ibm.com>
---
 ATLAS/CONFIG/src/atlcomp.txt |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: atlas/ATLAS/CONFIG/src/atlcomp.txt
===================================================================
--- atlas.orig/ATLAS/CONFIG/src/atlcomp.txt
+++ atlas/ATLAS/CONFIG/src/atlcomp.txt
@@ -199,7 +199,7 @@ MACH=POWER6 OS=ALL LVL=1010 COMPS=f77
 MACH=POWER5 OS=ALL LVL=1010 COMPS=f77
    'gfortran' '-mcpu=power5 -mtune=power5 -O3 -fno-schedule-insns -fno-rerun-loop-opt'
 MACH=POWER7 OS=ALL LVL=1010 COMPS=f77
-   'xlf' '-qtune=pwr7 -qarch=pwr7 -O3 -qmaxmem=-1 -qfloat=hsflt'
+   'gfortran' '-O2 -m64 -mvrsave -funroll-all-loops'
 MACH=POWER5 OS=ALL LVL=1010 COMPS=f77
    'xlf' '-qtune=pwr5 -qarch=pwr5 -O3 -qmaxmem=-1 -qfloat=hsflt'
 MACH=POWER4 OS=ALL LVL=1010 COMPS=icc,dmc,smc,dkc,skc,xcc,gcc
